packages/a/asciinema/package.yml

```diff
@@ -1,22 +1,28 @@
 name       : asciinema
-version    : 2.4.0
-release    : 14
+version    : 3.0.0
+release    : 15
 source     :
-    - https://github.com/asciinema/asciinema/archive/refs/tags/v2.4.0.tar.gz : b0e05f0b5ae7ae4e7186c6bd824e6d670203bb24f1c89ee52fc8fae7254e6091
+    - https://github.com/asciinema/asciinema/archive/refs/tags/v3.0.0.tar.gz : f44feaa1bc150e7964635dc4714fd86089a968587fed81dccf860ee7b64617ca
 homepage   : https://asciinema.org/
 license    : GPL-3.0-or-later
 component  : multimedia.video
 summary    : Terminal session recorder and the best companion of asciinema.org
 description: |
     Terminal session recorder and the best companion of asciinema.org
+networking : true
 builddeps  :
-    - pkgconfig(python3)
-    - python-build
-    - python-installer
-    - python-setuptools
-    - python-wheel
+    - rust
+environment: |
+    export ASCIINEMA_GEN_DIR=$installdir/usr/share
```

Comment:
It would probably be better to generate them in a directory in the current directory, like `dist` or something, and install them into the install root from there.